### PR TITLE
#2950 add myanmar auth string translations

### LINKED ISSUE #2950: Add Burmese/Myanmar auth strings
## Is your feature request related to a problem? Please describe.

Add Burmese/Myanmar translations for auth strings.

## Describe the solution you'd like

See above.

## Implementation

Add Burmese/Myanmar translations to `authStrings` localisation.

## Describe alternatives you've considered

N/A.

## Additional context

See #2928 for main issue.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #2950,

## Change summary

Adds Burmese/Myanmar auth string translations.

## Testing

Not sure how to go about this, as don't think we have anyone on team who can speak Burmese!

Instead of writing exhaustive tests for all translations, might be easier to do a test run with a user?

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Login menu translations are correct.
- [ ] On updating sync details in settings, warning message is correctly translated.

### Related areas to think about

N/A.

### CODE REVIEW

File: src/localization/authStrings.json [hunk 1]
```diff
@@ -39,6 +39,17 @@
     "password": "ລະຫັດຜ່ານ",
     "user_name": "ຊື່​ຜູ້​ນຳ​ໃຊ້"
   },
+  "my": {
+    "logging_in": "Login ဝင်ရောက်နေသည်",
```
Comment (mark-prins) [inline src/localization/authStrings.json:43]: actually, the `Login` here is redundant. Google translate has `ဝင်ရောက်နေသည်` == `Signing in`

File: src/localization/authStrings.json [hunk 2]
```diff
@@ -39,6 +39,17 @@
...
+    "login": "Login ဝင်ရောက်သည်",
```
Comment (mark-prins) [inline src/localization/authStrings.json:44]: the `Login` isn't required, afaik

Comment (wlthomson) [reply src/localization/authStrings.json:44]: You sure? Took it directly from the the translations... maybe should confirm with @AungHtunOo88?

Comment (AungHtunOo88) [reply src/localization/authStrings.json:44]: We just started using standard Unicode for our country recently. So, some google translation may not come up perfectly but with general usage (e.g - log-in). But, we can confirm that our translation will be easily understood by the basic user.

**Pull request merged**